.github/workflows/e2e-wpuf.yml

```
@@ -0,0 +1,122 @@
+
+      
+      #######--- Remove Later ---#######
+      - name: Publish Test Report
+        uses: mikepenz/action-junit-report@v3
+        if: always() # always run even if the previous step fails
+        with:
+          report_paths: "tests/e2e/playwright-report/results.xml"
+          include_passed: true,
+          # check_retries: true,
+          # detailed_summary: true,
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)